pkg/daemon/daemon.go

```diff
@@ -376,13 +395,18 @@ func (d *daemon) DeletePeriodicUpdate() {
 			}
 		}
 
-		for _, guidAddr := range guidList {
+		for index, guidAddr := range guidList {
 			if err = d.guidPool.ReleaseGUID(guidAddr.String()); err != nil {
+				failedPods = append(failedPods, passedPods[index])
```

Comment:
what is passedPods? it seem weird that you add the passedPods to the failedPods

Reply:
The pods that are configured with IB and then passed the remove pkey in delete periodic, Then failed in releasing guid, I am not sure if the is case may happen,
do u want to discard this change?